examples/stack/values-xl.yaml

```diff
@@ -39,6 +39,8 @@ metrics:
                   - {key: kubernetes.io/os, operator: NotIn, values: [windows]}
 
     agent:
+      prom_config:
+        host_filter: "true"
```

Comment:
The template is looking for `.Values.prom_config.host_filter`, but this change would add `.Values.agent.prom_config.host_filter`. These two lines need to be moved out of the agent block -- you can put it just under `grafana-agent` before the `controller` block.